app/views/users/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<h1>New user</h1>
+
+<!--
+<form action="/users" method="post" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<% form_authenticity_token %>">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br/><br/>
+  <label for="email">Email:</label>
+  <input type="text" id="email" name="user[email]"><br/><br/>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br/><br/>
+  <input type="submit" value="Submit">
+</form>
+-->
+
+<!--
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag(:username) %><br/><br/>
+  <%= label_tag(:email, "Email") %>
+  <%= text_field_tag(:email) %><br/><br/>
+  <%= label_tag(:password, "Password") %>
+  <%= password_field_tag(:password) %><br/><br/>
+  <%= submit_tag "Submit" %>
+<% end %>
+-->
+
+<%= form_for @user do |f| %>
+  <%= f.label :username, "Username:" %><br />
+  <%= f.text_field :username, placeholder: "Username" %>
+  <% @user.errors.full_messages_for(:username).each do |message| %>
+  <div><%= message %></div>
+  <% end %>
+  <br /><br />
+  <%= f.label :email, "Email:" %><br />
+  <%= f.text_field :email, placeholder: "Email" %>
+  <% @user.errors.full_messages_for(:email).each do |message| %>
+  <div><%= message %></div>
+  <% end %>
+  <br /><br />
+  <%= f.label :password, "Password:" %><br />
+  <%= f.password_field :password, placeholder: "Password" %>
+  <% @user.errors.full_messages_for(:password).each do |message| %>
+  <div><%= message %></div>
+  <% end %>
+  <br /><br />
+  <%= f.submit %>
+<% end %>
```

Comment:
Use `#form_with` here as it was instructed in the [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms) project.